docker-compose.yaml

```diff
@@ -22,11 +22,16 @@ services:
       - SERVERFULL_RUNTIME_STATS_OUTPUT=NULL
       - SERVERFULL_RUNTIME_SIGNALS_INSTALLED=OS
       - SERVERFULL_RUNTIME_SIGNALS_OS_SIGNALS=15 2
-      - "POSTGRES_PASSWORD=password"
-      - "POSTGRES_USERNAME=user"
-      - "POSTGRES_DATABASENAME=assetmgmt"
-      - "POSTGRES_HOSTNAME=postgres"
-      - "POSTGRES_PORT=5432"
+      - "AIAPI_POSTGRES_PASSWORD=password"
```

Comment:
question:
what is the motivation to add the prefix `AIAPI_` to these `POSTGRES` environment variables? Why is it also not needed above in the `docker-compose.it.yml`?

Reply:
Because docker-compose.it.yml uses the code that does not include top-level component for configuration and reads env vars directly (see code in tests/*). I am not  100% confident this is the right thing to do, but that's the way it is now.

Reply:
Not 100% confident - not re: the service, but re: the tests. The prefix is a well established pattern in the rest of our services.